doctordroid/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
+    "manifest_version": "2.0.0",
+    "app_uuid": "5e75658c-065e-460f-b9f8-42bf100e361d",
+    "app_id": "doctordroid",
+    "display_on_public_website": true,
+    "tile": {
+      "overview": "README.md#Overview",
+      "configuration": "README.md#Setup",
+      "uninstallation": "README.md#Uninstallation",
+      "support": "README.md#Support",
+      "changelog": "CHANGELOG.md",
+      "description": "Analyse your alerts to identify trends to improve noise and coverage",
+      "title": "Doctor Droid",
+      "media": [
+        {
+          "media_type": "image",
+          "caption": "Datadog Alert Occurence Trend",
+          "image_url": "images/alert_occurence_count_trends.png"
+        },
+        {
+          "media_type": "image",
+          "caption": "Datadog Top 5 alerts occurence distribution",
+          "image_url": "images/top_5_alerts_distribution.png"
+        },
+        {
+          "media_type": "image",
+          "caption": "Top 10 lifetime alert types and affected services",
+          "image_url": "images/top_10_lifetime_alert_counts.png"
+        }
+      ],
+      "classifier_tags": [
+        "Supported OS::Linux",
+        "Supported OS::macOS",
+        "Supported OS::Windows",
+        "Category::Automation",
+        "Category::Incidents",
+        "Offering::UI Extension"
```

Comment:
Please also add the types of data that this integration either submits to Datadog, or pulls from Datadog. For example:

```
"Queried Data Type::Metrics",
"Submitted Data Type::Metrics"
```

You can view the full list of options [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags).

Reply:
Added